project/standard/templates/configs/pluginsConfig.json

```diff
@@ -139,6 +139,17 @@
       "children": ["Permalink"],
       "autoEnableChildren": ["Permalink"]
     },
+    {
+      "name": "DynamicLegend",
+      "glyph": "align-left",
+      "title": "plugins.DynamicLegend.title",
+      "description": "plugins.DynamicLegend.description",
+      "dependencies": [
+        "Toolbar",
+        "BurgerMenu",
+        "SidebarMenu"
+      ]
```

Comment:
In https://github.com/allyoucanmap/MapStore2/commit/a11b6736dd0cc5175977ec75a212d3c7d205d309 only the SidebarMenu is included as dependency because is the current preferred container for all side plugins